Containers/mastercontainer/Dockerfile

```diff
@@ -109,31 +99,23 @@ RUN sed -i \
         echo "ServerName localhost" | tee -a /etc/apache2/httpd.conf; \
         echo "LoadModule ssl_module modules/mod_ssl.so" | tee -a /etc/apache2/httpd.conf; \
         echo "LoadModule socache_shmcb_module modules/mod_socache_shmcb.so" | tee -a /etc/apache2/httpd.conf; \
-        echo "Include /etc/apache2/sites-available/mastercontainer.conf" | tee -a /etc/apache2/httpd.conf
-
-RUN set -ex; \
+        echo "Include /etc/apache2/sites-available/mastercontainer.conf" | tee -a /etc/apache2/httpd.conf; \
+    \
     rm -f /etc/apache2/conf.d/default.conf \
           /etc/apache2/conf.d/userdir.conf \
-          /etc/apache2/conf.d/info.conf
-
-RUN mkdir /var/log/supervisord; \
+          /etc/apache2/conf.d/info.conf; \
+    \
+    mkdir /var/log/supervisord; \
     mkdir /var/run/supervisord;
 
-COPY Caddyfile /
-COPY start.sh /usr/bin/
-COPY backup-time-file-watcher.sh /
-COPY session-deduplicator.sh /
-COPY cron.sh /
-COPY daily-backup.sh /
-COPY supervisord.conf /
-COPY healthcheck.sh /
-RUN chmod +x /usr/bin/start.sh; \
-    chmod +x /cron.sh; \
-    chmod +x /session-deduplicator.sh; \
-    chmod +x /backup-time-file-watcher.sh; \
-    chmod +x /daily-backup.sh; \
-    chmod a+r /Caddyfile; \
-    chmod +x /healthcheck.sh
+COPY --chmod=664 Caddyfile /Caddyfile
+COPY --chmod=775 start.sh /usr/bin/start.sh
+COPY --chmod=775 backup-time-file-watcher.sh /backup-time-file-watcher.sh
+COPY --chmod=775 session-deduplicator.sh /session-deduplicator.sh
+COPY --chmod=775 cron.sh /cron.sh
+COPY --chmod=775 daily-backup.sh /daily-backup.sh
+COPY --chmod=664 supervisord.conf /supervisord.conf
+COPY --chmod=775 healthcheck.sh /healthcheck.sh
```

Comment:
can we put all of them into / and simply use one step for all of them by using an asterisk?

Reply:
Caddyfile and the supervisord probably need a step on their own

Reply:
done